bob/readme.md

```diff
@@ -350,22 +347,22 @@ For more information, please visit the [Flashbots Protect and MEV-Share Order Fl
 
 To subscribe to transactions, connect to the server:
 ```
-https://mevshare.bob.flashbots.net:443
+https://tx.tee-searcher.flashbots.net
 ```
 
 To submit bundles, connect to the server:
 ```
-https://ofa.bob.flashbots.net:443
+https://backruns.tee-searcher.flashbots.net
 ```
 
 
 Please run the following command to **cache DNS resolution** (remember: DNS is not available during production mode!)
 
 ```bash
-echo "3.149.14.12 mevshare.bob.flashbots.net" >> /etc/hosts
-echo "3.136.107.142 mevshare.bob.flashbots.net" >> /etc/hosts
-echo "18.221.59.61 ofa.bob.flashbots.net" >> /etc/hosts
-echo "3.15.88.156 ofa.bob.flashbots.net" >> /etc/hosts
+echo "3.149.14.12 tx.tee-searcher.flashbots.net" >> /etc/hosts
+echo "3.136.107.142 tx.tee-searcher.flashbots.net" >> /etc/hosts
+echo "18.221.59.61 backruns.tee-searcher.flashbots.net" >> /etc/hosts
+echo "3.15.88.156 backruns.tee-searcher.flashbots.net" >> /etc/hosts
```

Comment:
optional: could this be moved to `/usr/bin/init-container.sh` so that searchers don't need to run this manually?